From 0346c7039d9448b897db07e5c72a11c314ec1394 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Steven!=20Ragnar=C3=B6k?=
 <nuclearsandwich@users.noreply.github.com>
Date: Mon, 23 Nov 2020 15:39:47 -0800
Subject: [PATCH 1/2] Add idlc dependencies to release package.xml. (#7)

Adding this solely to the debian branches to preserve the existing
behavior of not building idlc / depending on java / maven in from-source
builds.
---
 package.xml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/package.xml b/package.xml
index 727dece..cd4ebef 100644
--- a/package.xml
+++ b/package.xml
@@ -13,6 +13,8 @@
     <url type="repository">https://github.com/eclipse-cyclonedds/cyclonedds</url>
 
     <buildtool_depend>cmake</buildtool_depend>
+    <buildtool_depend>java</buildtool_depend>
+    <buildtool_depend>maven</buildtool_depend>
 
     <depend>openssl</depend>
     <test_depend>libcunit-dev</test_depend>
-- 
2.29.2

